ReSharper.FSharp/src/FSharp.Psi/src/Impl/Cache2/Parts/FSharpTypeMembersOwnerTypePart.cs

```diff
@@ -96,5 +98,29 @@ public override IEnumerable<ITypeElement> GetSuperTypeElements()
       SuperTypesClrTypeNames = superTypeNames.ToArray();
       return superTypeElements;
     }
+
+    public override IEnumerable<ITypeMember> GetTypeMembers()
+    {
+      var declaration = GetDeclaration();
+      if (declaration == null)
+        return EmptyList<ITypeMember>.Instance;
+
+      var result = new LocalList<ITypeMember>();
+      foreach (var memberDeclaration in declaration.MemberDeclarations)
+      {
+        var declaredElement = memberDeclaration.DeclaredElement;
+        if (declaredElement is FSharpProperty<MemberDeclaration> prop)
+        {
+          if (prop.Getter != null) result.Add(prop.Getter);
+          if (prop.Setter != null) result.Add(prop.Setter);
+        }
+        else
+        {
+          result.Add(declaredElement);
+        }
+      }
+
+      return result.ResultingList();
```

Comment:
This method duplicates the base one almost completely.

Reply:
not anymore (:

Reply:
Yes, now it should probably belong to `FSharpTypeElementDeclarationBase.ProcessChildren`. It should also take declared elements from accessors declarations, not from the property declared element.

Reply:
Still not resolved.